pkgs/development/python-modules/filedate/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pip,
+  python-dateutil,
+}:
+buildPythonPackage rec {
+  pname = "filedate";
+  version = "3.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "kubinka0505";
+    repo = pname;
```

```suggestion
    repo = "filedate";
```
`repo` and `pname` are semantically different, so we should not use `repo = pname`.

see also https://github.com/NixOS/nixpkgs/issues/277994

---

Updated